schemas/dab.draft.schema.json

```diff
@@ -367,11 +367,11 @@
             },
             "log-level": {
               "type": "object",
-              "description": "Global configuration of log level",
+              "description": "Global configuration of log level, defines logging severity levels for specific classes, when 'null' it will set logging level based on 'host: mode' property",
               "additionalProperties": false,
               "patternProperties": {
                 "^[a-zA-Z_]\\w{0,}(\\.\\w{1,}){0,}$": {
-                  "description": "Defines logging severity levels for specific classes, when 'null' it will set logging level based on 'host: mode' property",
+                  "description": "Regex pattern checks that it only allows for namespace values. So it only accepts a letter as its first value, from there it accepts any alphanumeric character, each section is divided by '.' and there cannot be more than 1 '.' in a row",
```

Comment:
Even though the Regex explains checking for namespaces, we should mention that only specific namespaces (classes) are valid here. Since we don't support adding any generic namespace name.

Reply:
```suggestion
                  "description": "Regex pattern checks that it only allows for namespace values. So it only accepts a letter as its first value, from there it accepts any alphanumeric character, each section is divided by '.' and there cannot be more than 1 '.' in a row. Additionally, only valid DataApiBuilder namespaces are allowed.",
```